server/db/User.js

```diff
@@ -14,30 +14,46 @@ class User extends Model {
       firstName: {
         type: STRING,
         allowNull: false,
-        validate: {
-          len: [2, 100]
-        }
+        validate: { len: [2, 100] }
       },
       lastName: {
         type: STRING,
         allowNull: false,
-        validate: {
-          len: [2, 100]
-        }
+        validate: { len: [2, 100] }
       },
-      office: {
+      password: {
         type: STRING,
         allowNull: false,
-        validate: {
-          is: office
+        validate: { notEmpty: true },
+        private: true
+      }
+    };
+    const hooks = {
+      beforeCreate(user) {
+        return password.hash(user.password)
+          .then(hash => user.password = hash);
+      },
+      beforeUpdate(user) {
+        if (user.changed('password')) {
```

Comment:
Swap condition and use early return instead. :bowtie:

Reply:
I was wondering should I do that, since I didn't see the point of returning anything, or even doing anything since there is only one action to do if the condition is met but I suppose it's the 'tabs' thing.

Reply:
It is exactly that _tabs_ thing 😉